README.md

```diff
@@ -1,80 +1,14 @@
-# OpenTelemetry Instrumentation: Spring and SpringBoot
+# OpenTelemetry Instrumentation: Spring and Spring Boot
+<!-- ReadMe is in progress -->
+<!-- TO DO: Add sections for starter guide -->
 
-This package streamlines the manual instrumentation process in OpenTelemetry for Spring and SpringBoot. It will enable you to add traces to requests, and database calls with minimal changes to application code. This package will not fully automate your OpenTelemetry instrumentation, instead, it will provide you with better tools to instrument your own code.
-This contribution to OpenTelemetry will follow in the footsteps of the existing Spring integration in [OpenCensus](https://github.com/census-instrumentation/opencensus-java/tree/master/contrib/spring/src/main/java/io/opencensus/contrib/spring).
 
-This starter guide contains three tutorials. The first tutorial will walk you through span creation and propagation using the JavaSDK and the HTTP client RestTemplate. 
+This package streamlines the manual instrumentation process in OpenTelemetry for [Spring](https://spring.io/projects/spring-framework) and [Spring Boot](https://spring.io/projects/spring-boot) applications. It will enable you to add traces to requests, and database calls with minimal changes to application code. This package will not fully automate your OpenTelemetry instrumentation, instead, it will provide you with better tools to instrument your own code.
 
-The second tutorial will build on the first. It will walk you through implementing spring-web handler and interceptor interfaces to create traces without modifying your application code. 
+This starter guide contains three sections. In the first two sections you will use the "vanilla" Open Telemetry API. In the third section, we will explore features defined in this package and use these tools to better instrument your application. 
 
-The third tutorial will walk you through the annotations and configurations defined in the opentelemetry-contrib-spring package. This tutorial will equip you with new tools to streamline the step up and configuration of OpenTelemetry on Spring and SpringBoot applications.
+The first section will walk you through span creation and propagation using the JavaSDK and the HTTP web client, RestTemplate. 
 
-To use the tools included in this package include the dependency below in your spring application:
+The second section will build on the first. It will walk you through implementing spring-web handler and interceptor interfaces to create traces with minimal changes to your application code. 
```

Comment:
[Note] I thought the "minimal changes to your application code" was going to be the selling point for the third section which is your contribution? 
Or is that a different facet of the OpenTelemetry API?

Reply:
I will be using the handlers presented in section 2 to create the annotations presented in section 3. I'm kind of taking the process one step future to remove the need of copy and pasting java files.